examples/oneapi/cpp/BUILD

```diff
@@ -8,6 +8,9 @@ dal_module(
     name = "example_util",
     hdrs = glob(["source/example_util/*.hpp"]),
     includes = [ "source" ],
+    dal_deps = [
+        "@onedal//cpp/oneapi/dal/algo:jaccard",
```

Comment:
jaccard?

Reply:
Yes, it looks strange, to build, e.g., PCA example, I'll need to build jaccard